go.mod

```diff
@@ -1,20 +1,21 @@
 module sigs.k8s.io/cluster-api-provider-openstack
 
-go 1.13
+go 1.15
 
 require (
-	github.com/go-logr/logr v0.1.0
-	github.com/gophercloud/gophercloud v0.8.0
-	github.com/gophercloud/utils v0.0.0-20200302155035-0565566533e4
-	github.com/onsi/ginkgo v1.12.1
-	github.com/onsi/gomega v1.10.1
+	github.com/go-logr/logr v0.3.0
+	github.com/gophercloud/gophercloud v0.15.0
+	github.com/gophercloud/utils v0.0.0-20201203161420-f41c1768a042
+	github.com/onsi/ginkgo v1.14.1
+	github.com/onsi/gomega v1.10.2
 	github.com/pkg/errors v0.9.1
+	github.com/spf13/pflag v1.0.5
 	gopkg.in/yaml.v2 v2.3.0
-	k8s.io/api v0.17.9
-	k8s.io/apimachinery v0.17.9
-	k8s.io/client-go v0.17.9
+	k8s.io/api v0.20.2
+	k8s.io/apimachinery v0.20.2
+	k8s.io/client-go v0.20.2
 	k8s.io/klog v1.0.0
-	k8s.io/utils v0.0.0-20200619165400-6e3d28b6ed19
-	sigs.k8s.io/cluster-api v0.3.12
-	sigs.k8s.io/controller-runtime v0.5.14
+	k8s.io/utils v0.0.0-20210111153108-fddb29f9d009
+	sigs.k8s.io/cluster-api v0.3.11-0.20210219155426-bc756c4e7ed0
```

Comment:
Why not v0.3.12?

Reply:
cluster-api and other providers are implementing v1alpha4 on master and keep older release series on release branches (like our recently created release-0.3 branch)

3.14 is the latest cluster-api release on the release-0.3 branch and does not contain the v1alpha4 implementation.

Plan would be for us, to now depend on cluster-api master until they are releasing v0.4.0. After that we can start using their releases again except we need some latest changes on master then.